ckanext/protected_resources/plugin.py

```diff
@@ -68,7 +34,7 @@ def resource_delete(context, data_dict):
     # if you can update a package then you can delete it
     can_update = tk.check_access('package_update', context, pkg_dict)
```

Comment:
Is a package just another name for a dataset?

Reply:
Yes